tests/core/core.spec.ts

```diff
@@ -47,7 +47,7 @@ describe("@dataform/core", () => {
       const t2 = session
         .publish("schema2.example", {
           type: "table",
-          dependencies: ["schema1.example"],
+          dependencies: [{ schema: "schema1", name: "example" }],
```

Comment:
this is great. one last thing... I'm betting this doesn't work if the user requests schema suffixes? (in exactly the same way as referencing `dataset` in tests?)

Reply:
That's right it was not working. I have made some modifications and changed the test case `it("config",...`. Now it runs with and without a suffix. Now it should work